README.md

```diff
@@ -56,6 +56,7 @@ stacks-azure-data
 ├── docs            # Documentation
 ├── pysparkle       # Python library built to streamline data processing; packaged and uploaded to DBFS
 ├── utils           # Python utilities package used across solution for local testing
+├── .docstr.yaml    # Configuration for docstring coverage checks
```

Comment:
to be updated in docs as well (here and amido)

Reply:
done (and PR opened https://github.com/amido/amido.github.io/pull/354)